### PR TITLE
Skios 161 angular bookmarks buttons

### PR DESCRIPTION
**Pull request by jeppard**
# Skiosa - PR
- [ ] added test(s), or explain why one is not needed/possible...
- [x] added comments to all the functions
- [ ] added documentation to latex project
- [x] code can't be split up into smaller logical units
- [x] code is formated and linted

### CODE REVIEW

**Comment by simonmorgenstern** [comment]
resolve your own conflicts? :D

**Comment by jeppard** [comment]
Testing would be overly complicated, because the only added method is using apollo and keycloak sevice

**Comment by jeppard** [comment]
> resolve your own conflicts? :D

Yes, allready done

**Comment by simonmorgenstern** [comment]
resolve conflicts again? :P

**Comment by Y05h129** [comment]
Screenshot_20220509_201357
Ich kann das leider nicht starten, es wirft mir den Fehler siehe Screenshot

File: src/app/core/queries/bookmarks.ts [hunk 1]
```diff
@@ -21,6 +22,13 @@ export const BOOKMARKS_QUERY = gql<BOOKMARKS_QUERY_RESPONSE, { PaginationArg: Pa
 			categories {
 				id
 			}
+			bookmarkStatus
 		}
 	}
 `;
+
+export const CHANGE_BOOKMARK_MUTATION = gql<{ changeBookmark: boolean }, { isBookmarked: boolean; articleId: number }>`
```
Comment (simonmorgenstern) [inline src/app/core/queries/bookmarks.ts:30]: please add a short comment to explain params

**Review by simonmorgenstern — APPROVED** [review]
finally looks good to me :)

**Pull request merged**